### ISSUE TITLE
cull should be permissionless

### ISSUE THREAD

**Issue by hexonaut** [makerdao/MIP21-RWA-Example#4]
As a general rule of thumb the invocation of bad debt should not require governance to execute it as the incentives are not well aligned. We have a timeout already so I think it's fine to just allow anyone to call `cull`. I realize governance could indefinitely extend it, but I still think it's better to enforce a hard date that requires intervention to extend rather than requiring intervention to execute.

**Comment by SebVentures** [comment]
Agree if the timeout is big (but that defeat the point). But then governance is no longer able to mark down some investments before the timeout which is not ideal neither.

**Comment by hexonaut** [comment]
Yeah I'm not suggesting for the current MIP22. This just needs to be re-worked before we scale up.
